interpolator/interpolator.F90

```diff
@@ -742,30 +811,30 @@ subroutine interpolator_init( clim_type, file_name, lonb_mod, latb_mod, &
            & trim(adjustl(lowercase(file_calendar))) == 'julian')  .or. &
            & (model_calendar == NOLEAP .and.   &
            & trim(adjustl(lowercase(file_calendar))) == 'noleap') )  then
-        call mpp_error (NOTE, 'interpolator_mod: Model and file&
+        call mpp_error (NOTE, 'interpolator[2]_mod: Model and file&
```

Comment:
Why the change to the module name in the error messages?  The module is still interpolator_mod.  I find the '[2]' a distraction.